docs/test.ts

```diff
@@ -574,7 +574,7 @@ const enabled = llmobs.enabled
 // manually enable
 llmobs.enable({
   mlApp: 'mlApp',
-  agentlessEnabled: true
+  agentlessEnabled: false
```

Comment:
this is just for docs? Any reason we're changing this value on a docs snippet?

Reply:
while this file is just testing our types are valid TS when interpreted, it does actually do/execute everything in this file. i changed it because this call will start llmobs and try to start our writers, so we don't want the file to complain about a missing api key.